bin/docker-worker

```diff
@@ -1,19 +1,19 @@
 #!/bin/bash
 set -e
 
-if [ -z "$REDIS_URL" ]
+if [ -z "$REDIS_URL$POSTHOG_REDIS_HOST" ]
 then
   echo "⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️"
   echo "️⚠️ 🚨🚨🚨 PostHog warning! 🚨🚨🚨"
   echo "⚠️"
-  echo "️⚠️ The environment variable REDIS_URL is not configured!"
+  echo "️⚠️ The environment variable REDIS_URL or POSTHOG_REDIS_HOST is not configured!"
   echo "⚠️ Redis will be mandatory in the next versions of PostHog (1.1.0+)."
   echo "⚠️ Please configure it now to avoid future surprises!"
   echo "⚠️"
   echo "⚠️ See here for more information!"
   echo "⚠️ --> https://docs.posthog.com/#/upgrading-posthog?id=upgrading-from-before-1011"
   echo "⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️"
 else
-  celery -A posthog beat -S redbeat.RedBeatScheduler &
-  celery -A posthog worker
+  ./bin/docker-worker-beat &
```

Comment:
I'm not sure about splitting this up into _that_ many files but also not blocking for merging this

Reply:
Yeah, it's a lot, but in k8s we will run the beat and worker in separate pods... but for users who just do `docker run` and just get it over with, it'll all be in one command. So it kind of makes sense... 🤷‍♂️ 

![image](https://user-images.githubusercontent.com/53387/79132597-eb129f80-7daa-11ea-9190-a768cd012fad.png)

Reply:
That's fine 👍